content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -0,0 +1,117 @@
+You can find their list in [command-line-tools-reference]. Example of klog format:
+
+```
+I1025 00:15:15.525108       1 httplog.go:79] GET /api/v1/namespaces/kube-system/pods/metrics-server-v0.3.1-57c75779f-9p8wg: (1.512ms) 200 [pod_nanny/v0.0.0 (linux/amd64) kubernetes/$Format 10.56.1.19:51756]
+```
+
+[command line tools reference]: /docs/reference/command-line-tools-reference
+[klog]: https://github.com/kubernetes/klog
+
+### Structured Logging
+
+{{<warning>}}
+Migration to structured log messages is an ongoing process. Not all log messages are structured in this version. When parsing log files, you must also handle unstructured log messages.
+
+Structured logging is an alpha feature. Log formatting and value serialization are subject to change.
+{{< /warning>}}
+
+Structured logging is a effort to introduce a uniform structure in log messages allowing for easy extraction of information, making logs easier and cheaper to store and process.
+New message format is backward compatible and enabled by default.
+
+Format of structured logs:
+```
+<klog header> "<message>" <key1>="<value1>" <key2>="<value2>" ...
+```
+
+Example:
+```
+I1025 00:15:15.525108       1 controller_utils.go:116] "Pod status updated" pod="kube-system/kubedns" status="ready"
+```
+
+
+### JSON log format
+
+The `--logging-format=json` flag changes format of logs from components to JSON format.
+
+{{<warning >}}
+JSON output does not support many standard klog flags. The [command line tools reference] lists unsupported klog flags.
+
+Not all logs are guaranteed to be written in JSON format (for example, during process start). If you intend to parse logs, make sure you can handle logs in non-JSON formats.
+
+Structured logging is an alpha feature. Field names and JSON serialization are subject to change.
+{{< /warning >}}
+
+Example of JSON log format (pretty printed):
+```json
+{
+   "ts": 1580306777.04728,
+   "v": 4,
+   "msg": "Pod status updated",
+   "pod":{
+      "name": "nginx-1",
+      "namespace": "default"
+   },
+   "status": "ready"
+}
+```
+
+Keys with special meaning:
+* `ts` - timestamp as Unix time (required, float)
+* `v` - verbosity (required, int, default 0)
+* `err` - error string (optional, string)
+* `msg` - message (required, string)
+
+
+List of components currently supporting JSON format:
+* {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+* {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}}
+* {{< glossary_tooltip term_id="kube-scheduler" text="kube-scheduler" >}}
+* {{< glossary_tooltip term_id="kubelet" text="kubelet" >}}
+
+### Log verbosity level
+
+The `-v` flag controls log verbosity. Increasing the value increases the number of logged events. Decreasing the value decreases the number of logged events.
+Increasing verbosity settings logs increasingly less severe events. A verbosity setting of 0 logs only critical events.
```

Comment:
Since we are already here, can you help address issue #21404 by add a few lines?

Reply:
Like explained in comment above, there is no real limit. Currently in k/k/ codebase is between 0-12. 

Outside of guide https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md (which looks outdated) there is no other documented meaning between verbosity levels. Guide only accounts for 0-5, but there are still around 200 log calls with verbosity between 6-12.

Reply:
Thanks for the detailed info. I'm copying your comment to that issue.